### PR TITLE
[Docs] [3.2.0] Fixes #2483 (Update graphQL rate limiting pages)

### LINKED ISSUE #2483: "GraphQL Rate Limiting" pages need to be reviewed
**Description:**
Quoting few below which are either grammatically incorrect, have punctuation mistakes or capitalizing issues etc.
Please go through the entire sub section as there are multiple mistakes.

[Overview](https://apim.docs.wso2.com/en/latest/learn/rate-limiting/graphql-api/overview-query-limits-for-graphql/) Page

Screen Shot 2020-12-08 at 4 21 43 PM
--
Screen Shot 2020-12-08 at 4 23 59 PM

[Query Depth Limit](https://apim.docs.wso2.com/en/latest/learn/rate-limiting/graphql-api/query-depth-limitation/) Page
Screen Shot 2020-12-08 at 4 27 00 PM

---
Screen Shot 2020-12-08 at 4 27 52 PM
-----
Screen Shot 2020-12-08 at 4 31 24 PM

**Affected Product Version:**
3.2.0

### PR DESCRIPTION
**Pull request by Chamindu36**
## Purpose
Fixing the capitalization and grammar issues in pages related to the GraphQL rate limiting

## Goals
Fixes https://github.com/wso2/docs-apim/issues/2483

## Approach
**/rate-limiting/graphql-api/overview-query-limits-for-graphql/ page**

screenshot-localhost-8000-learn-rate-limiting-graphql-api-overview-query-limits-for-graphql-1608539323469

**/rate-limiting/graphql-api/query-complexity-limitation/  page**
Screenshot from 2020-12-21 14-05-22

**/rate-limiting/graphql-api/query-depth-limitation/  page**
screenshot-localhost-8000-learn-rate-limiting-graphql-api-query-complexity-limitation-1608539729103

### CODE REVIEW

File: en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-computation cost of resolving the particular field**. 
+the computation cost of resolving the particular field**. 
 
 <html>
 <div class="admonition note">
 <p class="admonition-title">Note</p>
 <p>
 If the field needs to call an expensive service to resolve itself, 
-then the **complexity should be at a high level** but if the field is easy to resolve and not an expensive operation, 
+then the complexity should be at a high level but, if the field is easy to resolve and not a cheap operation,
```
Comment (uvindra) [inline en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md:16]: This should read as,

`then the complexity should be at a high level, but if the field is easy to resolve(an inexpensive operation),`

File: en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
 If the field needs to call an expensive service to resolve itself, 
-then the **complexity should be at a high level** but if the field is easy to resolve and not an expensive operation, 
+then the complexity should be at a high level but, if the field is easy to resolve and not a cheap operation,  
 the **complexity should be at a low level**. If **no complexity defined** for a field, **by default it will get a value of 1.** 
 </p>
 </div> 
 </html>
 
-In run time, requested query complexity value will be calculated and blocked it exceeds the max complexity.
+In the run time, the requested query complexity value will be calculated and, blocked if the value exceeds the max complexity.
```
Comment (uvindra) [inline en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md:22]: This should read as,

`At runtime, a query will be blocked if its calculated complexity exceeds the specified max complexity.`

File: en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
 
 Following is a simple example which can be used to understand how the calculation happens.
 
-If there are, no arguments provides in your requested query, can calculate complexity value simply by
-adding all the complexity values.
+If there are, no arguments provided in your requested query,  the complexity can be calculated simply by adding all the complexity values.
```
Comment (uvindra) [inline en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md:26]: This should read as,

`If there are no arguments provided in a query,  the total complexity can be calculated by simply adding the individual complexity values of each field.`

File: en/docs/learn/rate-limiting/graphql-api/query-depth-limitation.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Since GraphQL schemas often have circular relationships, the depth can grow without bounds. 
-this means a client could craft a query as follows:
+This means a client could craft a query as follows:
```
Comment (uvindra) [inline en/docs/learn/rate-limiting/graphql-api/query-depth-limitation.md:2]: This should read as,

`An example of such a query is as follows:`

**Review by uvindra — CHANGES_REQUESTED** [review]

File: en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md [hunk 5]
```diff
@@ -1,30 +1,29 @@
...
+Often, limiting only the depth of a query is not sufficient to protect a GraphQL service from complex queries. 
+The reason for this is that some fields in a GraphQL schema being more costly to compute than others. WSO2 API-Manager introduces 
 **“Query Complexity Limitation”** to address such cases. 
 
 With this strategy, a request allowed or rejected based on the complexity of the query, and the configured max complexity 
 value of the subscription policy for the corresponding API.
 
 Here we introduced the **complexity values for each of the Fields** in the schema. That **describes the 
-computation cost of resolving the particular field**. 
+the computation cost of resolving the particular field**.
```
[the remark this answers is not in the document]
Comment (Chamindu36) [reply en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md:9]: Fixed

**Review by uvindra — APPROVED** [review]

**Pull request merged**